### PR TITLE
Bonnie v1.3 list valueset overload

### PR DESCRIPTION
**Pull request by losborne**
Add support for new signature for In (Valueset)
```
in(codes List<Code>, valueset ValueSetRef) Boolean
```

https://jira.mitre.org/browse/BONNIE-1719
https://github.com/HL7/cql/blob/master/spec/09-b-cqlreference.adoc#128-in-valueset

Pull requests into cql-execution require the following.
Submitter and reviewer should ✔ when done.
For items that are not-applicable, mark "N/A" and ✔.

[CDS Connect](https://cds.ahrq.gov/cdsconnect) and [Bonnie](https://github.com/projecttacoma/bonnie) are the main users of this repository. 
It is strongly recommended to include a person from each of those projects as a reviewer.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Code diff has been done and been reviewed (it does not contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass ( 9 unrelated failing tests left on base branch )
- [x] Code coverage has not gone down and all code touched or added is covered.
(error condition is untested, but can't figure out how to get null to translate to a valueset for that condition.)
- [x] All dependent libraries are appropriately updated or have a corresponding PR related to this change N/A
- [x] regression test run and any regressions logged to jira (no regressions vs https://github.com/cqframework/cql-execution/commit/669ed2c302fc607e46222fcbdef08735781dd079)

**Reviewer:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: src/elm/clinical.coffee [hunk 1]
```diff
@@ -27,6 +27,24 @@ module.exports.ValueSetRef = class ValueSetRef extends Expression
...
+
+  exec: (ctx) ->
+    valueset = @valueset.execute(ctx)
+    # If the value set reference cannot be resolved, a run-time error is thrown.
+    throw new Error("ValueSet must be provided to InValueSet function") unless valueset? and valueset.isValueSet
+
+    codes = @codes.exec(ctx)
+    for code in codes
+      # If the code argument is null, the result is false.
+      return false unless code?
```
Comment (daco101) [inline src/elm/clinical.coffee]: I see the spec says that if the code argument is null, the result is false. But I think at that point the spec is referring to when the argument is one code, not a list. Are you sure it makes sense to return false if any code in the list is null? I think it might be better to ignore the null codes.
